debian/postinst

```diff
@@ -72,6 +72,29 @@ configure)
 		echo "ERROR: postgres id $(id -u postgres) != 65437" >&2
 		exit 1
 	fi
+
+  # We are switching to MAC address based network interface names ini AWS environment
+  # from 6.0.10.0 onwards. To handle this change, we leverage the default
+  # 80-net-setup-link.rules udev rules file. We make a copy of this file under /etc
+  # and then change the naming policy to MAC address based names. Please see DLPX-75209
+  # for more context on this change.
+  platform_type=$(cat /var/lib/delphix-appliance/platform)
```

Comment:
can we use the `get-appliance-platform` script instead?
```
$ get-appliance-platform 
aws
```

e.g.
```
if [[ "$(get-appliance-platform)" == "aws" ]]; then
```

Reply:
Yes, this looks much better and has been fixed.